Backpack/src/main/java/net/skyscanner/backpack/horisontalnav/BpkHorizontalNav.kt

```diff
@@ -0,0 +1,116 @@
+package net.skyscanner.backpack.horisontalnav
+
+import android.content.Context
+import android.content.res.ColorStateList
+import android.text.Spannable
+import android.text.SpannableStringBuilder
+import android.util.AttributeSet
+import androidx.annotation.AttrRes
+import androidx.annotation.ColorRes
+import com.google.android.material.tabs.TabLayout
+import net.skyscanner.backpack.R
+import net.skyscanner.backpack.text.BpkFontSpan
+import net.skyscanner.backpack.text.BpkText
+import net.skyscanner.backpack.util.*
+import net.skyscanner.backpack.util.ResourcesUtil
+import net.skyscanner.backpack.util.use
+import net.skyscanner.backpack.util.wrapContextWithDefaults
+
+open class BpkHorizontalNav @JvmOverloads constructor(
+  context: Context,
+  attrs: AttributeSet? = null,
+  defStyleAttr: Int = 0
+) : TabLayout(wrapContextWithDefaults(context), attrs, defStyleAttr) {
+
+  private enum class Appearance(
+    val id: Int,
+    @AttrRes val styleAttribute: Int,
+    @ColorRes val defaultTextColor: Int,
+    @ColorRes val defaultTextSelectedColor: Int,
+    @ColorRes val defaultIndicatorColor: Int
+  ) {
+    Normal(
+      id = 0,
```

Comment:
Can we use the Enum's `ordinal` value instead of adding an id prop? 
e.g `Normal.ordinal` will return `0`